Source/Plugins/IsacPluginGenerator/Assets/Microsoft.SpatialAudio.Spatializer.Unity/.npmrc

```diff
@@ -1,3 +1,3 @@
-registry=https://pkgs.dev.azure.com/aipmr/MixedReality-Unity-Packages/_packaging/Unity-packages/npm/registry/
+registry=https://microsoft.pkgs.visualstudio.com/Analog/_packaging/MixedReality-UPM-Internal/npm/registry/

```

Comment:
It looks like the name of the feed includes the word "Internal".  Is this a staging feed before the package goes to an external feed?  It's obvious that I don't know how all of this works.

Reply:
In the packaging stage we publish to an internal feed, which can be then tested to make sure everything is good. Once the build is "approved" after the packaging phase, the publishing phase then pushes the plugin to both nuget and UPM public feeds.